Forms/ItemsForm.cs

```diff
@@ -0,0 +1,340 @@
+                            item.Stocks -= 1;
+
+                            if (_purchaseItemManager.Add(purchaseItem) && _itemManager.Update(item))
+                            {
+                                _purchases.Add(new PurchaseDetails(purchaseItem, item));
+                                LoadData();
+                                CartForm cartForm = Application.OpenForms.OfType<CartForm>().FirstOrDefault();
+                                cartForm.LoadData();
+                            }
+                            else
+                            {
+                                string caption = "Can't add item(s).";
+                                string message = "Please try again.";
+                                MessageBox.Show(message, caption, MessageBoxButtons.OK);
+                                return;
+                            }
+                        }
+
+                        scope.Complete();
+                    }
+                }
+                else
+                {
+                    string caption = "No item selected.";
+                    string message = "Please select an item to be added in your cart.";
+                    MessageBox.Show(message, caption, MessageBoxButtons.OK);
+                }
+            }
+            catch (Exception ex)
+            {
+                Logger.log.Error(ex.ToString());
+            }
+        }
+
+        private void AddItemButton_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                Item item = new Item();
+                Item returnedItem = ShowDialogNewItem(item);
+
+                if (returnedItem != null)
+                {
+                    if (_itemManager.Add(returnedItem))
+                    {
+                        LoadData();
+                    }
+                    else
+                    {
+                        string caption = "Error.";
+                        string message = "Can't add item. Please try again.";
+                        MessageBox.Show(message, caption, MessageBoxButtons.OK);
+                    }
+                }
+            }
+            catch (Exception ex)
+            {
+                Logger.log.Error(ex.ToString());
+            }
+        }
+
+        private void ItemListView_ItemChecked(object sender, ItemCheckedEventArgs e)
+        {
+            try
+            {
+                int id = e.Item.SubItems[1].Text.ToInt(-1);
+
+                if (_purchases.Find(x => x.PurchaseItem.ItemId == id) != null || _itemManager.GetById(id).Stocks == 0)
+                {
+                    e.Item.Checked = false;
+                }
+            }
+            catch (Exception ex)
+            {
+                Logger.log.Error(ex.ToString());
+            }
+        }
+
+        private Item ShowDialogNewItem(Item item)
+        {
+            try
+            {
+                Form prompt = new Form();

```

Comment:
You can just create a form for this one. No need to create the form programatically.